samples/subsys/event_manager/src/events/ack_event.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _ACK_EVENT_H_
+#define _ACK_EVENT_H_
+
+/**
+ * @brief ACK Event
+ * @defgroup ack_event ACK Event
+ * @{
+ */
+
+#include "event_manager/event_manager.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct ack_event {
+	struct event_header header;
+};
+
+EVENT_TYPE_DECLARE(ack_event);
+
+#ifdef __cplusplus
+}
+#endif
```

Comment:
does this need to run on C++ code?

Reply:
I expect sample simply copied code from some previously defined event. Is there any harm coming out of it?